.circleci/config.yml

```diff
@@ -60,6 +60,17 @@ jobs:
           paths:
             - ".git"
 
+  lint:
```

Comment:
Looks good.

I was surprised it didn't show up on the PR. But then realised it is missing a section like this: https://github.com/CosmWasm/wasmd/pull/645/files#diff-78a8a19706dbd2a4425dd72bdab0502ed7a2cef16365ab7030a5a0588927bf47L194-L196

Can you add 3 lines to workflow (running after setup-dependencies makes sense to me), and I can see the results in the CI.